packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -12,8 +12,11 @@ import 'platform_interface.dart';
 import 'src/webview_android.dart';
 import 'src/webview_cupertino.dart';
 
+/// Optional callback invoked when a [WebView] is first created. [controller] is
+/// the [WebViewController] for the created WebView.
```

Comment:
just a nit - WebView became camelcase, but I think it refers to "webview" the widget, not "WebView" the class

Reply:
Done.